src/app/features/search/search.component.ts

```diff
@@ -23,7 +23,7 @@ export class SearchComponent {
     this.globalErrorService.errors = [];
     this.searchError = undefined;
     if (searchTerm) {
-      this.technicalRecordService.searchBy({ type: 'vin', searchTerm });
+      this.technicalRecordService.searchBy({ type: 'vin', searchTerm: searchTerm.trim() });
```

Comment:
`trim()` should be done on or before checking value of `searchTerm` as `'    '` would still evaluate to `true` and send an empty string to `searchBy` method.

Reply:
Thank you